app/controllers/api/v2/courses_controller.rb

```diff
@@ -143,6 +144,8 @@ def update_course
 
         @course.assign_attributes(course_params)
         @course.save
+
+        Courses::UpdateNotificationService.new.call(course: @course)
```

Comment:
Would this line be better after line 98 instead?

Reply:
Since the service needs to be called after a given save I'd like to keep it near the actual call to `.save()`

Reply:
sure, i guess out of scope of this PR this controller action could do with a tidy up